README.md

```diff
@@ -196,6 +199,55 @@ Explicitly:
 pytest -p no:nbmake
 ```
 
+## "By Cell" Mode
```

Comment:
```suggestion
## Use your notebook as an integration test suite (experimental 🧪)
```

I'd like to frame this as a user story so that potential users understand why they should use this mode. (We may want to consider renaming).